src/components/CallView/CallView.vue

```diff
@@ -75,36 +75,14 @@
 							:shared-data="sharedDatas[shownRemoteScreenPeerId]"
 							is-big />
 						<!-- presenter overlay -->
-						<VueDraggableResizable v-if="shouldShowPresenterOverlay"
-							:key="presenterOverlaySize"
-							parent
-							:resizable="false"
-							:h="presenterOverlaySize"
-							:w="presenterOverlaySize"
-							:x="10"
-							:y="10">
-							<VideoVue class="presenter-overlay__video"
```

Comment:
This class is not moved as well

Reply:
What do you mean? it is in PresenterOverlay

Reply:
Maybe I miss it then. Now it's all good